### PR TITLE
Python 3.9 build - checked + works

### PR DESCRIPTION
**Pull request by piotrmaslanka**
Prepared a working image for OpenSIM 4.1 for Python 3.9

## Description
You can just type:
```bash
python
import opensim
```
and it works. I've also added GitHub Actions to the repository to further ensure that our future changes don't break the build.

## Motivation and Context
Further modernizing our software.

## How has this been tested?
In no way. @jimmybuffi told me that `import opensim` was sufficient to 95% possibility.
 

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Enhancement (enhances currently existing functionality or changes it in some way that is non-breaking and also not addressing a bug)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] My code requires a new environment variable or other globally accessible value.

### CODE REVIEW

**Comment by jimmybuffi** [comment]
We've found that if you open a python console and import opensim, that is 95% of the battle!

Ultimately, we need to test this by running inverse kinematics on a movement and check if that works, but if you can successfully import opensim that is a very good sign

**Comment by piotrmaslanka** [comment]
Yeah, I'm working on converting it to a full pull request once I can confirm that it builds on my machine.

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright 2020 Reboot Motion, Inc.
+Copyright 2020-2022 Reboot Motion, Inc.
```
Comment (piotrmaslanka) [inline LICENSE:1]: License was fixed free of charge.

**Comment by piotrmaslanka** [comment]
Closing due to bad repo

**Comment by RobertCalise** [comment]
Closing this since Python 3.9 and OpenSim 4.1 are not compatible.

**Pull request closed without merging**